xbmc/DynamicDll.cpp

```diff
@@ -83,6 +85,16 @@ bool DllDynamic::EnableDelayedUnload(bool bOnOff)
   return true;
 }
 
+bool DllDynamic::SetSystemLibrary(bool isSystem)
```

Comment:
I followed the line of this one:

bool DllDynamic::EnableDelayedUnload(bool bOnOff)

Same situation -> should I remove the return value also?
RetVal seems to be never used also.